### PR TITLE
Implement standard mySociety footer on whatdotheyknow.com

### LINKED ISSUE #310: Implement standard mySociety footer on whatdotheyknow.com
Moving this from https://github.com/mysociety/sysadmin/issues/676, where there is lots of background and discussion.

WhatDoTheyKnow does have the mySociety logo top right and 'WDTK is a mySociety project' bottom left; however the introduction of a new footer would make the affiliation more prominent.

### PR DESCRIPTION
**Pull request by zarino**
Fixes #310.

A pretty simple implementation of the [standard mySociety footer](https://github.com/mysociety/standard-footer) module.

While I was there, I also lightened the footer font colour, to pass WCAG AA.

screen shot 2016-05-05 at 15 35 38

### CODE REVIEW

**Comment by lizconlan** [comment]
:+1: looks good, rebase onto master might be messy tho 😿

**Comment by lizconlan** [comment]
> rebase onto master might be messy tho 😿

I take that back, it's not too bad, waiting for Vagrant box to spin up so I can check I've not broken the design with my rebase attempt (retaining the onclick analytics code from master, dumping everything else)

File: lib/views/general/_responsive_footer.html.erb [hunk 1]
```diff
@@ -1,43 +1,66 @@
...
+
+        <div class="col-sm-3">
+            <ul class="mysoc-footer__badges">
+                <li role="presentation"><a href="https://github.com/mysociety/whatdotheyknow-theme" class="mysoc-footer__badge mysoc-footer__badge--github">Github</a></li>
+              <% unless AlaveteliConfiguration::twitter_username.blank? %>
+                <li role="presentation">
+                  <a href="https://twitter.com/<%= AlaveteliConfiguration::twitter_username %>" class="mysoc-footer__badge mysoc-footer__badge--twitter" onclick="<%= track_analytics_event(AnalyticsEvent::Category::OUTBOUND, AnalyticsEvent::Action::TWITTER_EXIT, :label => 'Footer link')%>">@<%= AlaveteliConfiguration::twitter_username %></a>
+                </li>
+              <% end %>
+                <li role="presentation"><a href="https://facebook.com/whatdotheyknowcom" class="mysoc-footer__badge mysoc-footer__badge--facebook" onclick="<%= track_analytics_event(AnalyticsEvent::Category::OUTBOUND, AnalyticsEvent::Action::FACEBOOK_EXIT, :label => 'Footer link')%>">Facebook</a></li>
```
Comment (garethrees) [inline lib/views/general/_responsive_footer.html.erb]: Looks like this should have a `AlaveteliConfiguration::facebook_username` conditional

Comment (lizconlan) [inline lib/views/general/_responsive_footer.html.erb]: D'oh, lost in the rebase 🔨

**Pull request merged**